custom/icds_reports/templates/icds_reports/icds_app/mobile/navigation.directive.html

```diff
@@ -1,43 +1,189 @@
 {% load staticfiles %}
 <div class="sideNavigation" id="nav-menu">
-    <img src="{% static "icds_reports/mobile/images/white-close-icon.png" %}" class="close-icon" ng-click="closeMenu()">
-    <p class="navigation-font" ng-bind="userFullName" ng-if="userFullName"></p>
-    <p class="navigation-font mb-5" ng-bind="userUsername"></p>
-    <a class="menu-options" href="{$ goToStep('', $location.search()) $}" ng-click="closeMenu()">
-        <img src="{% static "icds_reports/mobile/images/program-summary.png" %}" class="mr-3">
-        <p class="navigation-font mb-0">Program Summary</p>
-    </a>
-    <div class="menu-options">
-        <img src="{% static "icds_reports/mobile/images/mother.png" %}" class="mr-3">
-        <p class="navigation-font mb-0">Maternal and Child Nutrition</p>
+    <img src="{% static 'icds_reports/mobile/images/white-close-icon.png' %}" class="close-icon" ng-click="closeMenu()">
+    <p class="navigation-font pl-3 pr-3 mb-0" ng-bind="userFullName" ng-if="userFullName"></p>
+    <p class="navigation-font pl-3 pr-3 mb-4" ng-bind="userUsername"></p>
+    <div style="overflow: auto; height: calc(100vh - 100px)">
+        <a class="menu-options" href="{$ goToStep('', $location.search()) $}" ng-click="closeMenu()">
+            <img src="{% static 'icds_reports/mobile/images/program-summary.png' %}" class="mr-3">
+            <p class="navigation-font mb-0">Program Summary</p>
+        </a>
+        <div class="menu-options" ng-click="expandedSectionId='nutrition'">
```

Comment:
how much of this is shared with the web navigation? this is probably out of scope for this PR, but is it plausible to imagine making `navigation-section.directive` and `navigation-item.directive` and then reusing the same data structure to create both sets of menus at some point in the future?

Reply:
I'm thinking of making a constant (array of objects with menu names and corresponding routes) in one of the shared utils and iterate over it in both web and mobile navigation.
Let me know your take on this.

Reply:
That sounds great, but can we do it in a follow up so we can demo this in the meantime?